rcl/include/rcl/time.h

```diff
@@ -323,7 +323,7 @@ rcl_fini_duration(rcl_duration_t * duration);
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_time_source_t *
-rcl_get_default_ros_time_source(void);
+rcl_default_ros_time_source_get(void);
```

Comment:
In fact all of the "`rcl_get_default_`..." function should be left alone I think. I'll try to comment on each of them below (at least in this header file).

You can think of it as a C++ function and it's clearer, i.e. this one would be:

```c++
rcl_time_source_t * rcl::get_default_ros_time_source()
```

You could also imagine it being `rcl_ros_time_source_get_default()` which in C++ would be like:

```c++
rcl_time_source_t * rcl::ros_time_source::get_default()
```

Where `rcl::ros_time_source` would be a class and `get_default` would be a static member function. But since both are equally valid, I'd prefer to leave it as-is.